File: event-service/src/main/kotlin/com/greenlight/eventservice/domain/EventParticipant.kt
package com.greenlight.eventservice.domain

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table

@Table
data class EventParticipant(
    @Id
    val id: Long? = null,
    var eventId: Long,
    var userId: String?,
    var userName: String? = null
)

File: gateway/src/main/webapp/green-light-web/src/app/domain/event/event-participant.ts
export class EventParticipant {
  id: number;
  eventId: number;
  userId: string;
  userName: string;
}

export class EventParticipantRequest {
  eventId: number;
  userId: string;

  constructor(eventId: number, userId: string) {
    this.eventId = eventId;
    this.userId = userId;
  }
}
